scipio/src/timer/mod.rs

```diff
@@ -6,4 +6,22 @@
 //! scipio::timer is a module that provides timing related primitives.
 mod timer_impl;
 
+use std::time::Duration;
 pub use timer_impl::{Timer, TimerActionOnce, TimerActionRepeat};
+
+/// Sleep for some time.
+///
+/// ```
+/// use scipio::LocalExecutor;
+/// use scipio::timer::sleep;
+/// use std::time::Duration;
+///
+/// let ex = LocalExecutor::make_default();
+///
+/// ex.run(async {
+///     sleep(Duration::from_millis(100)).await;
+/// });
+/// ```
+pub async fn sleep(wait: Duration) {
```

Comment:
write Unit Test that ensures default Executor actually sleeps while waiting

Reply:
new test is `test_no_spin` : `env SCIPIO_TEST_POLLIO_ROOTDIR=/tmp cargo test executor::test_no_spin -- --nocapture`